gadget/gadget.go

```diff
@@ -293,7 +293,7 @@ type VolumeStructure struct {
 	// ID is the GPT partition ID, this should always be made upper case for
 	// comparison purposes.
 	ID string `yaml:"id" json:"id"`
-	// Filesystem used for the partition, 'vfat', 'ext4' or 'none' for
+	// Filesystem used for the partition, 'fat16', 'vfat', 'ext4' or 'none' for
```

Comment:
my thinking is that the smallest and consistent change would be to support:

vfat, vfat-32, vfat-16  

where vfat and vfat-32 are synonyms

my preference for having a dash is because 32,16 here are parameters and not a version.

Maybe I'm reading the doc wrongs but it doesn't seem that choosing different bit size disables vfat extensions?